controls/radaiprompt/aiprompt-button.md

```diff
@@ -0,0 +1,45 @@
+---
+title: AIPrompt Button
+page_title: AIPrompt Button
+description: Check our &quot;AIPrompt Button&quot; documentation article for the RadAIPrompt control.
+slug: radaiprompt-prompt-button
+tags: prompt button, button, radaiprompt
+published: True
+position: 5
+---
+
+# AIPrompt Button
+
+The AIPrompt component can also be displayed as a button that will show a `RadAIPrompt` when interacted with it. To display the component as a button, create a new `RadAIPromptButton` instance.
+
+The RadAIPromptButton provides the following properties:
+
+* `AIPrompt`&mdash;This property allows you to set an RadAIPrompt instance that will be displayed when the RadAIPromptButton is open.
```

Comment:
You don't need "This property" part in the descriptions. You can start directly with "Sets the RadAIPrompt..."